### ISSUE TITLE
Implemented models

### ISSUE THREAD

**Issue by Anne-ClaireFouchier** [Anne-ClaireFouchier/MLforAR#13]

**Comment by Anne-ClaireFouchier** [comment]
EficientNet 

https://blog.tensorflow.org/2020/03/higher-accuracy-on-vision-models-with-efficientnet-lite.html

https://github.com/Ching-Chen-Wang/EfficientNet-eLite/blob/master/EfficientNet-eLite/confs/efficientnet_l1.yaml

https://github.com/Ching-Chen-Wang/EfficientNet-eLite/commit/6ca9cec3664c38c2ace9283f682acb4116a0e423


https://arxiv.org/pdf/2009.07409.pdf

**Comment by Anne-ClaireFouchier** [comment]
SSD Keras
https://github.com/pierluigiferrari/ssd_keras/blob/master/ssd300_inference.ipynb

https://towardsdatascience.com/implementing-ssd-in-keras-part-i-network-structure-da3323f11cff

**Comment by Anne-ClaireFouchier** [comment]
SSD + EfficientNet
https://github.com/CasiaFan/SSD_EfficientNet
https://github.com/shaoshengsong/EfficientNet-SSD/blob/master/train.py

**Comment by Anne-ClaireFouchier** [comment]
https://www.tensorflow.org/lite/examples/object_detection/overview

**Comment by Anne-ClaireFouchier** [comment]
https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md
